cmd/vic-machine/create/create.go

```diff
@@ -239,6 +241,12 @@ func (c *Create) Flags() []cli.Flag {
 			Usage:       "Virtual Container Host x509 certificate file",
 			Destination: &c.cert,
 		},
+		cli.StringFlag{
+			Name:        "base-image-size",
```

Comment:
Is this accurate? I am a little confused by the terminology here. Aren't we specifying the storage for images here?